### ISSUE TITLE
Check SMath build

### ISSUE THREAD

**Issue by ibell** [CoolProp/CoolProp#1106]
@mikekaganski I fought with cmake for a while this evening and I think the build is working now.  Can you please confirm that the generated binaries operate properly.  They should be up at http://www.coolprop.dreamhosters.com/binaries/SMath/ in a few minutes

**Comment by mikekaganski** [comment]
Ok, will do at once

**Comment by mikekaganski** [comment]
It works and tells me to update the module for HELMHOLTZMASS/MOLAR :)

**Comment by mikekaganski** [comment]
Also, in the package, there is a 6.0.0.0 directory, and coolprop_wrapper.dll file that is obsolete. CoolPropWrapper.dll should be kept. I guess, it remained in a directory from previous builds.

**Comment by ibell** [comment]
Make sure you wait for the updated build.

**Comment by mikekaganski** [comment]
I'm not sure, but it seems that you reverted my attempt to avoid placing generated files to source directory?

**Comment by mikekaganski** [comment]
Ok, I had files dated 2016-06-07 00:07

**Comment by mikekaganski** [comment]
Yes, now I have files from 14:04, and they indeed work!

**Comment by mikekaganski** [comment]
get_global_param_string("gitrevision")="ffddbce6b12f1718f50eaf23a87274a0f8aa76e7"

**Comment by ibell** [comment]
Good, I got super frustrated trying to get out of source build working with
buildbot, and I gave up.  I agree it's better to have out of source build
if we can.  I can try again...
On Jun 6, 2016 10:23 PM, "Mike Kaganski" <notifications@github.com> wrote:

get_global_param_string("gitrevision")="ffddbce6b12f1718f50eaf23a87274a0f8aa76e7"

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/CoolProp/CoolProp/issues/1106#issuecomment-224171571>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABxha_Xr2HKgvQGhOJUQm7grBLEmlaUpks5qJPJKgaJpZM4Iviak>
.
